### DISCUSSION TITLE
How to overcome a bug in Firefox 50 with Babel?

### DISCUSSION THREAD

**Discussion by denisstasyev** [babel/babel#13706] [Q&A] [answered]
Hello everyone, could you suggest a solution to the problem with Firefox 50 bug? Here it is - the following code throw Error in FF50 ([Bugzilla's link](https://bugzilla.mozilla.org/show_bug.cgi?id=1101653)):
```js
for (const key in o) {...}
```
Error: `SyntaxError: missing = in const declaration`

I have tried to set **.browserslist** with `ie 10` (!), but it does not help me. I'm using **'@babel/preset-env'**, but unfortunately no transpiling to `var` occured. As I understand Babel believes that this is correct code, but how to solve this problem (to get rid of this error)?

**Comment by hzoo** [comment]
Do you have reproduction step with your config? Seems to work on the [repl ](https://babeljs.io/repl#?browsers=ie%2010&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=GYewTgBAFAxiB2BnALhA1gUwJ4QJbwhAEoIBvAXyA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=true&targets=&version=7.15.3&externalPlugins=&assumptions=%7B%7D) targeting ie 10

**Reply by denisstasyev** [reply]
It seems that things with the Babel are still good. I found that the problem connected with Rollup bundler used with Svelte plugin. Babel does not transpile generated code from Svelte.  I'll check my build config and come back.

**Comment by denisstasyev** [comment, marked as answer]
The problem was that some of the Svelte files have **.mjs** extension. Therefore, for their transpilation it's necessary to add **.mjs** into `extensions` property of [@rollup/plugin-babel](https://www.npmjs.com/package/@rollup/plugin-babel) (also don't forget to add **.js**, **.ts** and so on).
